about.html

```diff
@@ -0,0 +1,113 @@
+            </ul>
+        </div>
+        <div class="hamburger">
+            <ul>
+            <li class="bar"></li>
+            <li class="bar"></li>
+            <li class="bar"></li>
+        </ul>
+        </div>
+        <div  class="flex" >
+          <a  class="logo" href="index.html"><img  id="logo"  src="assets/Earth.png"/></a>
+          <ul class="nav-menu">
+            <li class="nav-item">
+            <a class="nav-link" id="nav1"  href="index.html">Home</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#">Program</a>
+          </li>
+          <li class="nav-item">
+              <a class="nav-link" href="#">Join</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#">Sponsor</a>
+            </li>
+          <li class="nav-item">
+            <a class="nav-link"  href="#">News</a>
+          </li>         
+          <li class="nav-item GE">
+              <a class="nav-link" id="ge" href="#">Campaign</a>
+            </li>    
+        </ul>
+        </div>
+    </nav>
+   <!--Header-->
+   <header>
+    <h1 class="heading1about">"Hello!Sharing world"</h1>
+    <h2 class="heading2">Green Earth Global Summit 2023</h2>
+    <p  class="paragabout">Human existence is quite impossible without the presence of a healthy ecosystem. <br/>Our environment comprises all living and non-living components and their interactions within a natural habitat.<br/> Environmental conservation has become one of the core issues that need to be addressed to battle climate change and global warming.Green Earth Global aims at conservation of natural resources.<br/>Promoting sustainable production of safer foods through cleaner, more efficient and cost effective technologies for improved livelihoods and national development. </p>
+      </header>
+
+<!-- more info -->
+  <section class="info-sec">
+    <p class="info-p">please contact us for further questions regarding our annual summit<br>
+        <a class="info" href="#">info@GreenEarth.org ></a>
+    </p>
+  </section>
+
+<!-- past summit -->
+<section class="past-summit">
+    <h3 class="summit">Green Earth Global Summit 2020</h3>
+    <hr class="hr">
+    <p class="majisafiP">The logo of Green Earth organisation was changed as of 21st July, 2020. We thank everyone who took part in voting for the logo below.</p>
+    <img class="majisafi" src="assets/Earth.png" alt="logo"/>
+</section>
+<hr class="line">
+<!-- Past pictures -->
+<section class="pastphotos">
+    <h4 class="pastp">See the past G.E Global summit </h4>
+    <hr class="hr">
+    <p class="pag"> Take a look at the past two G.E global summit which took place<br>in Buenos Aires and Warsaw.</p>
+    <div >
+        <ul class="group-photo">
+        <li >
+             <img class="grp" src="assets/group1.png" alt="group photo"/>
+                </li>
+        <li ><img class="grp" src="assets/group2.png" alt="group photo"/>
+            </li>
```

Comment:
- [ ] Please ensure that the number and texts are on top of the image. What I have noticed is that your texts are attached the image which may be hard to edit the texts. Kindly fix that to ensure that your application is professional 👍.